.travis.yml

```diff
@@ -76,6 +76,7 @@ before_script:
 script:
   - ./scripts/copy_opencv.sh
   - ./scripts/main_script.sh
+  - ./gradlew build jacocoTestReport connectedAndroidTest 
```

Comment:
As I mentioned this should go into the `main-script.sh` script and not her on `.travis.yml`